src/av2/evaluation/scene_flow/constants.py

```diff
@@ -0,0 +1,77 @@
+"""Constants for scene flow evaluation."""
+
+from enum import Enum, unique
+from typing import Dict, Final, List
+
+import av2.evaluation.scene_flow.eval as eval
+from av2.datasets.sensor.constants import AnnotationCategories
+
+SCENE_FLOW_DYNAMIC_THRESHOLD: Final = 0.05
+
+CATEGORY_TO_INDEX: Final = {**{"NONE": 0}, **{k.value: i + 1 for i, k in enumerate(AnnotationCategories)}}
+
+FLOW_METRICS: Final = {
+    "EPE": eval.compute_end_point_error,
+    "Accuracy Strict": eval.compute_accuracy_strict,
+    "Accuracy Relax": eval.compute_accuracy_relax,
+    "Angle Error": eval.compute_angle_error,
```

Comment:
These names end up as columns in the results dataframe. Since they're then used on the leaderboard and when you make plots etc, I think its better if they are capitalized normally.